google-cloud-workflows-executions-v1/.repo-metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "api_id": "workflowexecutions.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-workflows-executions-v1/latest",
+    "distribution_name": "google-cloud-workflows-executions-v1",
+    "language": "ruby",
+    "name": "workflowexecutions",
+    "name_pretty": "Workflows Executions V1 API",
+    "product_documentation": "https://cloud.google.com/workflows/",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "Workflows link series of serverless tasks together in an order you define. Combine the power of Google Cloud's APIs, serverless products like Cloud Functions and Cloud Run, and calls to external APIs to create flexible serverless applications. Workflows requires no infrastructure management and scales seamlessly with demand, including scaling down to zero.. Note that google-cloud-workflows-executions-v1 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-workflows instead. See the readme for more details.",
```

Comment:
Typo: `including scaling down to zero..`

This typo also appears in master branch in `google-cloud-workflows/.repo-metadata.json` and `google-cloud-workflows-executions-v1beta/.repo-metadata.json`.

Reply:
Thanks. I'll get that fixed upstream.